### PR TITLE
add tests for kdump_post/pre scripts

### PR DESCRIPTION
**Pull request by emmawu**
Signed-off-by: xiawu <xiawu@redhat.com>

### CODE REVIEW

File: lib/gen-helper-script [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    shbang="msh"
+else
+    shbang="sh"
+fi
+
+root=$(mount | awk '{ if ($3 == "/") print $1; }')
+pre="kdump-pre"
+post="kdump-post"
+
+cat > /bin/kdump-pre.sh <<EOF
```
Comment (lulinqing) [inline lib/gen-helper-script]: how about using ${pre}.sh here as in line 16?
or hardcoded both if not necessary.

File: lib/gen-helper-script [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    mount --rbind \$dir \$tdir
+    mount -o remount,rw \$tdir
+fi
+echo "$pre.sh: kdump pre script." >\$tdir/root/$pre.log
+sync
+umount \$tdir
+sleep 5
+EOF
+
+cat > /bin/kdump-post.sh <<EOF
```
Comment (lulinqing) [inline lib/gen-helper-script]: $post.sh

Comment (lulinqing) [inline lib/gen-helper-script]: ${post}.sh as pre

Comment (emmawu) [inline lib/gen-helper-script]: Thanks. Updated both kdump-pre/post.sh

**Pull request merged**